src/apps/service/apps/agave/listings.clj

```diff
@@ -29,14 +29,14 @@
 (defn get-app-details
   [agave app-id admin?]
   (-> (.getAppDetails agave app-id)
-      (add-agave-job-stats admin?)
+      (add-agave-job-stats () admin?)
```

Comment:
Usually we just pass `nil` as the empty value. That's also safe for the `add-date-limits-where-clause` destructuring (`start_date` and `end_date` would just be `nil` also).

Reply:
Ah, good point. 👍